changelog/5201.bugfix.rst

```diff
@@ -0,0 +1,9 @@
+Fix bug encountering:
```

Comment:
This will mess with the log rendering, the first line should be short. e.g.
```
Fixed incorrectly raised Error encountered in pipelines with a ``ResponseSelector`` and NLG.

...
```